ui/src/react-admin/modules/shared/components/PaginationBar/PaginationBar.module.scss

```diff
@@ -0,0 +1,84 @@
+// @use "src/styles/abstracts" as *;
```

Comment:
Do we still need this? If so, uncomment this otherwise [kill it with fire ](https://giphy.com/clips/halloweenmovie-halloween-movie-2018-sE5nyUELuEM7qmDoMm)

Reply:
I used the PaginationBar from `react-components`. This component will be deleted.